editor/src/components/canvas/right-menu.tsx

```diff
@@ -54,8 +54,8 @@ export interface RightMenuTileProps extends React.HTMLAttributes<HTMLDivElement>
 export const RightMenuTile: React.FunctionComponent<RightMenuTileProps> = (props) => {
   const [hovered, setHovered] = React.useState(false)
 
-  const handleOnMouseOver = () => setHovered(true)
-  const handleOnMouseOut = () => setHovered(false)
+  const handleOnMouseOver = React.useCallback(() => setHovered(true), [])
```

Comment:
did this get smarter? nice!

Reply:
yeah, it detects if non-memoised callbacks are used even if they're defined outside of the JSX element. 👍